src/backend/highlighter.js

```diff
@@ -114,10 +137,12 @@ function getFragmentRect ({ _fragmentStart, _fragmentEnd }) {
  * @return {Rect}
  */
 
-const range = document.createRange()
 function getTextRect (node) {
-  range.selectNode(node)
-  return range.getBoundingClientRect()
+  if (!isBrowser) return
+  if (getTextRect.range) getTextRect.range = document.createRange()
```

Comment:
Perhaps we can create a variable outside of this function instead of assigning `range` to the function itself? It's a bit confusing atm